services/src/main/resources/uk/gov/moj/sdt/services/messaging/jms.properties

```diff
@@ -0,0 +1,39 @@
+########################################################################################################################
```

Comment:
Probably want to remove the corresponding jms.properties.template file (services/src/main/java/uk/gov/moj/sdt/services/messaging/jms.properties.template).  The jms.properties file was originally generated from the jms.properties.template file during the build process and it looks like this will no longer be the case.